package.json

```diff
@@ -21,7 +21,7 @@
     "karma": "^3.0.0",
     "karma-browserstack-launcher": "^1.3.0",
     "karma-chrome-launcher": "^2.2.0",
-    "karma-firefox-launcher": "^1.1.0",
+    "karma-firefox-launcher": "^1.2.0",
```

Comment:
This seems to solve #445 

It doesn't solve the failing assertion. I suggest we see that as a separate issue. This PR is strictly an improvement.

```
> snabbdom@0.7.3 test C:\Users\Dell\Documents\OpenSource\snabbdom
> karma start

04 09 2019 19:45:31.483:INFO [compiler.karma-typescript]: Compiling project using Typescript 3.3.4000
04 09 2019 19:45:36.949:INFO [compiler.karma-typescript]: Compiled 26 files in 5396 ms.
04 09 2019 19:45:37.839:INFO [bundler.karma-typescript]: Bundled imports for 26 file(s) in 387 ms.
04 09 2019 19:45:39.200:INFO [karma-server]: Karma v3.1.4 server started at http://0.0.0.0:9876/
04 09 2019 19:45:39.203:INFO [launcher]: Launching browsers Chrome, Firefox with concurrency unlimited
04 09 2019 19:45:39.210:INFO [launcher]: Starting browser Chrome
04 09 2019 19:45:39.226:INFO [launcher]: Starting browser Firefox
04 09 2019 19:45:40.819:INFO [Chrome 76.0.3809 (Windows 10.0.0)]: Connected on socket EGKSW_lJr-FnzFvdAAAA with id 58845090
................................................................................
...................................................................
Chrome 76.0.3809 (Windows 10.0.0): Executed 147 of 147 SUCCESS (1.03 secs / 0.851 secs)
04 09 2019 19:45:44.691:INFO [Firefox 69.0.0 (Windows 10.0.0)]: Connected on socket k-IMgjlfEwzfRTwKAAAB with id 78551669
................................................................................
.....................................................
Firefox 69.0.0 (Windows 10.0.0) style applies tranform as transition on remove FAILED
        uncaught exception: AssertionError: 1 === 0 (:0)
.............
Firefox 69.0.0 (Windows 10.0.0): Executed 147 of 147 (1 FAILED) (0.908 secs / NaN secs)
TOTAL: 1 FAILED, 293 SUCCESS
npm ERR! Test failed.  See above for more details.
```